source-code/ide-extension/src/services/telemetry/implementation.ts

```diff
@@ -38,6 +38,15 @@ async function capture(args: CaptureEventArguments) {
 	if (userID === undefined) {
 		userID = await getUserId()
 	}
+	if (args.event === "IDE-EXTENSION activated") {
+		telemetry.groupIdentify({
+			groupType: "repository",
+			groupKey: gitOrigin,
+			properties: {
+				name: gitOrigin,
+			},
+		})
+	}
```

Comment:
Same here. I would implement this directly after https://github.com/inlang/inlang/blob/84116139da02753ba3c69864ae439a3e43fc84e6/source-code/ide-extension/src/main.ts#L20-L27

Reply:
That was my first approach, but I had to import GitOrigin in this case. I assume that it is not so pretty to import the GitOrign here. 
But i will look into it again :)

Reply:
Ah fair. Looks cleaner regardless i would assume :)